.github/workflows/cargo-update.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  workflow_dispatch:
+    branches: [ main ]
+  schedule:
+    - cron: '15 3 * * 4'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+name: cargo-update
+jobs:
+  create-pr:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - name: Update dependencies
+        run: helper/bump-dependencies.sh 
+        shell: bash
```

Comment:
Let's avoid a helper script here. Just call `cargo update` four times with the appropriate `--manifest-path=$PATH/Cargo.toml` argument.

Reply:
Addressed